optashift-employee-rostering-webapp/pom.xml

```diff
@@ -46,6 +46,50 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-war-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>war-exploded</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>exploded</goal>
+            </goals>
+            <configuration>
+              <webappDirectory>${wildfly.home}/standalone/deployments/${project.build.finalName}.war</webappDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+            <!-- here the phase you need -->
+            <phase>package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${wildfly.home}/standalone/deployments/${project.build.finalName}.war/gwtui</outputDirectory>
+              <resources>          
+                <resource>
+                  <directory>${project.root}/optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/public</directory>
+                  <filtering>true</filtering>
```

Comment:
This PR enables filtering for those files (which can have dangerous consequences for binary files),
but I don't see changes in this PR to those files, to take advantage of that filtering?